assets/scss/components/hfg/frontend/layout/_general.scss

```diff
@@ -83,6 +83,9 @@ $desktop: map-get($gl-devices-list, desktop);
 [data-row-id] {
 	color: var(--color);
 	background: var(--bgColor);
+	border-bottom-width: var(--headerBorderWidth, 0);
+	border-bottom-color: var(--headerBorderColor, #000);
+	border-bottom-style: solid;
```

Comment:
Let's simplify this: 

```
border-bottom: var(--headerBorderWidth, 0) solid var(--headerBorderColor);
```

We should also use shorter variables: `--rowBWidth` & `--rowBColor`

We should always aim for the lightest CSS we can 🚀.